src/screens/TopicMemberScreen/index.js

```diff
@@ -31,13 +23,13 @@ const styles = StyleSheet.create({
     backgroundColor: colors.white
   },
   fragmentContentContainer: {
-    flexGrow: 1
+    // flexGrow: 1,
+    backgroundColor: 'white'
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Commenting out `flexGrow: 1` might affect the layout depending on its usage in the component. If it's not needed, it's better to remove it completely instead of commenting it out.
```diff
- flexGrow: 1
+ // flexGrow: 1,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->